pom.xml

```diff
@@ -75,6 +75,11 @@
 				-->
 			</exclusions>
 		</dependency>
+   		<dependency>
+      		<groupId>com.microsoft.azure</groupId>
+      		<artifactId>spring-data-cosmosdb</artifactId>
+     		 <version>2.2.2</version>
+    	</dependency>
```

Comment:
Need to fix alignment here.

Reply:
I changed the entire file to spaces instead of tabs so we would not have alignment issues - should be fixed. Going forward, I have VSCode set to show whitespaces in my code so will fix everything in an ongoing fashion.